packages/web/src/components/edit-playlist/desktop/EditPlaylistModal.tsx

```diff
@@ -62,9 +68,31 @@ const EditPlaylistModal = () => {
   const onCancelDelete = () => setShowDeleteConfirmation(false)
 
   const handleSubmit = useCallback(
-    (formFields: any) => {
+    (formFields: PlaylistFormValues, initialValues: PlaylistFormValues) => {
       if (playlistId) {
-        dispatch(editPlaylist(playlistId, formFields))
+        // Make what things changed
+        const metadataChanges = omitBy(formFields, (value, key) =>
+          isEqual(value, initialValues[key])
+        ) as Partial<PlaylistFormValues>
+        track({
```

Comment:
echoing what andrew said, do we need to track every metadata update for playlists? is this something we do for track edit? if not i might say not worth, and just collect the access changes like you're doing below.

Reply:
Yeah I think thats fair. We can defer to later if we want to track any specific changes